src/client/input.ts

```diff
@@ -25,8 +25,8 @@ export class Keyboard implements api.Keyboard {
     this._channel = channel;
   }
 
-  async down(key: string) {
-    await this._channel.keyboardDown({ key });
+  async down(key: string, overrides: channels.PageKeyboardDownOptions['overrides']) {
```

Comment:
`options: channels.PageKeyboardDownOptions`? Why doesn't the standard way work here?

Reply:
Because key is duplicated as the parameter name and as an override. I one of them should probably get renamed.